tsconfig.strict.json

```
@@ -48,6 +48,8 @@
     "lib/util/split.ts",
     "lib/util/url.ts",
     "lib/versioning/maven/**/*.ts",
+    "lib/versioning/rez/*.ts",
+    "lib/versioning/ruby/*.ts",
```

Just to make it more bullet-proof

```suggestion
    "lib/versioning/rez/**/*. ts",
    "lib/versioning/ruby/**/*.ts",
```